### PR TITLE
feat: completed contribute to source

### PR DESCRIPTION
**Pull request by Emmanuel-Develops**

### CODE REVIEW

**Comment by Emmanuel-Develops** [comment]
Closes #40

File: src/components/customSearchboxView/SearchInput.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  return (
+    <div className="search-box-with-contribute">
+      <div className="sui-search-box__wrapper">
+        <input {...getInputProps()} />
+        {getAutocomplete()}
+      </div>
+      <input {...getButtonProps()} />
+      <div className="search-box__contribute-wrapper">
+        <span role={"button"} onClick={openForm}>
+          Contribute to source?
```
Comment (adamjonas) [inline src/components/customSearchboxView/SearchInput.jsx:18]:
```suggestion
          Add a source
```

**Comment by adamjonas** [comment]
Looks good to ship with that copy change. Can we put a character limit on the URL so that someone doesn't stuff it with something ridiculous?

**Comment by Emmanuel-Develops** [comment]
> Looks good to ship with that copy change. Can we put a character limit on the URL so that someone doesn't stuff it with something ridiculous?

Alright. The max character limit for a domain is **255** char. I think it's safe to enforce that.

**Pull request merged**